ansible/README.md

```diff
@@ -0,0 +1,190 @@
+
+If you secured your server and you need to `sudo` for root access, use the the `-b` or `--become` option to tell ansible to sudo to root after connecting to the server. In the Terraform-DigitalOcean example, if you created the ec2-user by adding the `noroot=true` option (or if you are simply on Amazon AWS), you need to add the options `-u ec2-user -b` to ansible to tell it to connect as the ec2-user and then sudo to root to run the playbook.
+
+### DigitalOcean
+```
+DO_API_TOKEN="<The API token received from DigitalOcean>"
+TF_VAR_TESTNET_NAME="testnet-servers"
+ansible-playbook -i inventory/digital_ocean.py install.yml -e service=basecoin
+```
+
+### Amazon AWS
+```
+AWS_ACCESS_KEY_ID='<The API access key ID received from Amazon>'
+AWS_SECRET_ACCESS_KEY='<The API secret access key received from Amazon>'
+TF_VAR_TESTNET_NAME="testnet-servers"
+ansible-playbook -i inventory/ec2.py install.yml -e service=basecoin
+```
+
+### Installing custom versions
+
+By default ansible installs the tendermint, basecoin or ethermint binary versions from the latest release in the repository. If you build your own version of the binaries, you can tell ansible to install that instead.
+
+```
+GOPATH="<your go path>"
+go get -u github.com/tendermint/basecoin/cmd/basecoin
+
+DO_API_TOKEN="<The API token received from DigitalOcean>"
+TF_VAR_TESTNET_NAME="testnet-servers"
+ansible-playbook -i inventory/digital_ocean.py install.yml -e service=basecoin -e release_install=false
+```
+
+Alternatively you can change the variable settings in `group_vars/all`.
+
+## Other commands and roles
+
+There are few extra playbooks to make life easier managing your servers.
+
+* install.yml - Install basecoin or ethermint applications. (Tendermint gets installed automatically.) Use the `service` parameter to define which application to install. Defaults to `basecoin`.
+* reset.yml - Stop the application, reset the configuration and data, then start the application again. You need to pass `-e service=<servicename>`, like `-e service=basecoin`. It will restart the underlying tendermint application too.
+* restart.yml - Restart a service on all nodes. You need to pass `-e service=<servicename>`, like `-e service=basecoin`. It will restart the underlying tendermint application too.
+* stop.yml - Stop the application. You need to pass `-e service=<servicename>`.
+* start.yml - Start the application. You need to pass `-e service=<servicename>`.
+
+The roles are self-sufficient under the `roles/` folder.
+
+* install-tendermint - install the tendermint application. It can install release packages or custom-compiled binaries.
+* install - install the application defined in the `service` parameter. It can install release packages and update them with custom-compiled binaries.
+* rmdb - delete the tendermint database for a service.
```

Comment:
Does this also remove the ABCI app database? If it doesn't it would be very useful if it would. So something that resets only the data but leaves the configuration alone.